<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup rainfall data</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>380</width>
     <height>90</height>
    </rect>
   </property>
   <property name="title">
    <string>Data types</string>
   </property>
   <widget class="QRadioButton" name="rbUseASCgirdLayer">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>29</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Use ASCII grid files(.asc)[mm]</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbUseTextFileMRF">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>57</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Use mean areal rainfall text file(.mrf, .txt)[mm]</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>380</width>
     <height>60</height>
    </rect>
   </property>
   <property name="title">
    <string>Time setup</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>24</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rainfall time step[min] :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRainfallTimeStep">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>22</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>410</width>
     <height>550</height>
    </rect>
   </property>
   <property name="title">
    <string>Selected data</string>
   </property>
   <widget class="QTableWidget" name="dgvRainfallList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>16</y>
      <width>390</width>
      <height>480</height>
     </rect>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btReomveAll">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>510</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>380</width>
     <height>340</height>
    </rect>
   </property>
   <property name="title">
    <string>Search rainfall files</string>
   </property>
   <widget class="QLineEdit" name="txtFolderPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>27</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lstRFfiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>58</y>
      <width>360</width>
      <height>180</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btAddSelectedFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>298</y>
      <width>360</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btOpenRfFolder">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>27</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select folder</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>248</y>
      <width>360</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="txtSaveFileName">
     <property name="geometry">
      <rect>
       <x>109</x>
       <y>11</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>16</y>
       <width>100</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Save file name : </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSaveFile">
     <property name="geometry">
      <rect>
       <x>312</x>
       <y>10</y>
       <width>40</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>530</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>530</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
